README.md

```diff
@@ -1,2 +1,25 @@
-# franky
-Franky employee profile aggregator
+# Franky
+Franky - employee profile aggregator.
+
+### General concept
+Franky is the web-service to collect all open information about person 
+based on profile social networks such as:
+* GitHub
+* StackOverflow
+* Dockerhub
+* LinkedIn
+
+Service can be used by HR managers to find out information and 
+statistics about a potential job seekers.  
+Also job seekers could use Franky as a tool for generating formatted CV.
+ 
+##### Collected information include:
+* activity statistics for each social networks for the last year
+* active projects timeline for each social networks and in general
+* the most popular tags from all social networks
+* used technologies from all projects
```

Comment:
Thanks for the submission @ValeriaShestakova! I suppose that the two last points are pretty much the same and one can be excluded.

```suggestion
```